packages/api-openfin/notification.html

```diff
@@ -0,0 +1,11 @@
+<h5 id="message">Message</h5>
```

Comment:
Assuming that this page needs to be valid HTML, worth updating to something like this?

```html
<!DOCTYPE html>
<meta charset="utf-8">
<title></title>
<style>
  html {
    border: solid 2px black;
  }
</style>
<h5 id="message"></h5>
<script>
  function onNotificationMessage(message) {
    document.getElementById('message').innerText = message.title;
  }
</script>
```

Might be worth keeping the `h5` empty too, without the text `Message`?

```html
<h5 id="message"></h5>
```